go.mod

```diff
@@ -73,30 +73,34 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20210817132244-67c28690af52
```

Comment:
You need to update openshift/api version as well to v0.0.0-20210831091943-07e756545ac1

Reply:
the installer is bumping to v0.0.0-20210907212721-c9d6fcc62893 -- https://github.com/openshift/installer/pull/5198/files, which would pick up the api commits that @Prashanth684 mentioned